.github/workflows/build-release.yml

```diff
@@ -0,0 +1,41 @@
+## This is triggered by the publishing of a release, and will build the assets and attach them.
+
+name: On Release 
+
+on:
+  release:
+    types: [published]
```

Comment:
Only a question, maybe is better to add a manual workflow to attach the build artifacts before the publishing? The difference is not too big, but now, the user subscribed receives the notification of a new version, and the artifacts are not available. Is not too much time, 5 minutes more or less if the build goes well (there is a possibility that it has some kind of problem). 
And the Android version must be generated (and attached I suppose) out of this flow, so the user will see the Android version but nothing more.
Not a big problem, only wanted to notice it.